### PR TITLE
Corret link error:404

### PR DESCRIPTION
**Pull request by yuxiaobo96**
Signed-off-by: yuxiaobo <yuxiaobogo@163.com>

### CODE REVIEW

**Comment by yuxiaobo96** [comment]
/assign @tstromberg
THX.

**Comment by RA489** [comment]
ref https://github.com/kubernetes/minikube/issues/5936

File: site/content/en/docs/Contributing/releasing.en.md [hunk 1]
```diff
@@ -110,7 +110,7 @@ Verify release checksums by running `make check-release`
 
 ## Update docs
 
-If there are major changes, please send a PR to update <https://kubernetes.io/docs/setup/minikube/>
```
Comment (RA489) [inline site/content/en/docs/Contributing/releasing.en.md]: This link also pointing to the same page. no 404 here

Comment (yuxiaobo96) [reply site/content/en/docs/Contributing/releasing.en.md]: I understand what you mean, I just want to update it.

**Pull request merged**